docs-src/cloud/migration.md

```diff
@@ -0,0 +1,88 @@
+tags:
+  - temporal cloud
+  - migration
+  - multi-cluster replication
+---
+
+Migrating to Temporal Cloud from a self-hosted Cluster will have different requirements depending on your usage.
+This guide provides some guidance based on our experience helping customers of all sizes successfully migrate.
+
+### What to expect from a migration
+
+Depending on your Workflows’ requirements, the migration process may be as simple as changing a few parameters, or may call for more extensive code changes.
+There are two aspects to consider when migrating: your Temporal Client connection code and your Workflow Executions.
+Here’s a high-level overview of what you can expect:
+
+- **Update the Client connection code in your Worker:** Your Client connection code requires several small changes to ensure Temporal Cloud becomes responsible for new Workflow Executions.
+- **Migrate Workflow Executions:** There are different approaches for new, running, and completed Workflow Executions.
+  - **New Workflow Executions:** After you’ve updated your Client connection code, Temporal Cloud will automatically be responsible for all new Workflow Executions.
+  - **Running Workflow Executions:** Short-running Workflows can often be drained and then started again on Temporal Cloud. Long-running Workflows that cannot be drained and might require you to implement more code changes to pass the state of the currently running Workflow to Temporal Cloud.
+  - **Completed Workflow Executions:** Completed Workflow Execution History cannot yet be migrated to Temporal Cloud.
+
+### Updating Client connection code in your Workers
+
+Whether you’re self-hosting Temporal or using Temporal Cloud, you manage runtime of your code.
+To migrate your Workflows to Temporal Cloud, you need to change some parameters in the Client connection code, such as the endpoint’s hostname and port number.
+
+The changes needed to direct your Workflow to your Temporal Cloud
+Namespace are only a few lines of code, including:
+
+- [Add your SSL certificate and private key](/cloud/saml-intro) associated with your Namespace.
+- [Create a string](/cloud/namespaces-intro) to reference your cloud endpoint `(namespace.unique_id.tmprl.cloud:port)`.
+- [Connect to Temporal Cloud](/cloud/get-started) with your Client.
+- [Configure tcld, the Cloud CLI](/cloud/tcld/index), with the same address, Namespace, and
+  certificate used to create a Client through code.
+
+### Migrating your Workflow Executions
+
+A Temporal Cluster stores the complete Event History for the entire lifecycle of a
+Workflow Execution.
+To migrate from a self-hosted Temporal Cluster to Temporal Cloud, you need to take into account the current state, Event History, and any future expectations of your Workflow Executions.
+
+**New Workflows are automatically be executed on Temporal Cloud.**
+Once your new code is deployed, all new executions of your Workflows will occur automatically on Temporal Cloud instead of your self-hosted Cluster.
```

Comment:
"occur" is a little confusing, do you mean "all new executions of your Workflow will automatically run"?